config-staging.json

```diff
@@ -89,6 +89,46 @@
       "supportedBy": [],
       "type": "EVM"
     },
+    {
+      "coinApi": "evm",
+      "explorerUrl": "https://arbiscan.io/tx",
+      "feeCurrencies": [
+        "native"
+      ],
+      "identifiers": {
+        "chainId": 42161
+      },
+      "memos": false,
+      "name": "Arbitrum",
+      "nativeAsset": "ARB",
+      "networkTicker": "ARB",
+      "nodeUrls": [
+        "https://api.blockchain.info/arb/nodes/rpc"
+      ],
+      "shortName": "Arbitrum",
+      "supportedBy": [],
+      "type": "EVM"
+    },
+    {
+      "coinApi": "evm",
+      "explorerUrl": "https://optimistic.etherscan.io/tx",
+      "feeCurrencies": [
+        "native"
+      ],
+      "identifiers": {
+        "chainId": 10
+      },
+      "memos": false,
+      "name": "Optimism",
+      "nativeAsset": "OP",
```

Comment:
Isn't the native asset for Optimism actually `ETH.OP`?

Reply:
That sounds correct. OP is a governance token as far as I remember.